recipe/meta.yaml

```diff
@@ -35,20 +34,35 @@ test:
     - "\"%R%\" -e \"library('R6')\""  # [win]
 
 about:
-  home: https://github.com/wch/R6/
+  home: https://github.com/r-lib/R6/
   license: MIT
-  summary: The R6 package allows the creation of classes with reference semantics, similar to
-    R's built-in reference classes. Compared to reference classes, R6 classes are simpler
-    and lighter-weight, and they are not built on S4 classes so they do not require
-    the methods package. These classes allow public and private members, and they support
-    inheritance, even when the classes are defined in different packages.
+  summary: Creates classes with reference semantics, similar to R's built-in reference classes.
+    Compared to reference classes, R6 classes are simpler and lighter-weight, and they
+    are not built on S4 classes so they do not require the methods package. These classes
+    allow public and private members, and they support inheritance, even when the classes
+    are defined in different packages.
   license_family: MIT
 
 extra:
   recipe-maintainers:
+    - conda-forge/r
```

Comment:
You can remove me and leave only the `r` team n PRs like that.
I do have access to all feedstocks anyway ;-p

Reply:
Ok, fine. Done :)

Reply:
Thanks!